packages/api/tests/models/test-rules-model.js

```diff
@@ -224,3 +225,51 @@ test.serial('it does not delete event source mappings if they exist for other ru
   await rules.delete(rule);
   await rules.delete(ruleThree);
 });
+
+test.serial('Creating a kinesis rule where an event source mapping already exists, but is not enabled, succeeds', async (t) => {
+  process.env.messageConsumer = randomString();
+
+  const item = fakeRuleFactoryV2({
+    workflow,
+    rule: {
+      type: 'kinesis',
+      value: randomString()
+    },
+    state: 'ENABLED'
+  });
+
+  const lambdaBefore = aws.lambda;
+  try {
+    aws.lambda = () => ({
```

Comment:
Is there a reason to not use `sinon` here?

Reply:
This was quicker and, when [CUMULUS-1207](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1207) is addressed, this stub is going to be removed anyway.

Reply:
Fair enough, but we don't know when CUMULUS-1207 will get worked, and in the interim I'd prefer to be aligned with our stubbing approach in other tests. If for no other reason than consistency.

Would you mind updating to use `sinon.stub(aws, 'createEventSourceMapping').resolves({ UUID: randomString() });`? If it doesn't work for some reason, we don't need to worry about it

Reply:
Changed in a6bac67ab1cc91f236fe9dfa244df1dc9c625ae7